lib/src/chip/filter-button/filter-button.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, HostBinding, Input, OnChanges } from '@angular/core';
```

Comment:
After seeing what the filter-button is exactly. I'm thinking this would be better left as a template style guide. The reason is that I don't think it's difficult to implement and I feel it **steps out-of-bounds for what I'd expect this component to do**